CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Unreleased
 
+## [0.8.0](https://github.com/stm32-rs/stm32-eth/tree/v0.8.0)
+* Update version of `smoltcp` to `v0.12.0` ([#99])
```

Comment:
At risk of commenting on something you're already fixing: this needs a link. (Ref the other PR links)

Reply:
```suggestion
* Update version of `smoltcp` to `v0.12.0` ([#99])

[#99]: https://github.com/stm32-rs/stm32-eth/pull/99
```